On The Map/Controller/StudentsLocationTableViewController.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  TableViewController.swift
+//  On The Map
+//
+//  Created by Michal Hus on 2/24/20.
+//  Copyright © 2020 Michal Hus. All rights reserved.
+//
+
+import UIKit
+
+class StudentsLocationTableViewController: UITableViewController {
+    
+    var students: [Student] = []
+    
+    @IBAction func refreshStudentsLocation(_ sender: Any) {
+        Client.getStudentsLocation { (response, error) in
+            guard let response = response, error == nil else {
+                return
+            }
+            self.refetchLocations(response: response)
+        }
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.tableView.reloadData()
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(true)
```

Comment:
same here with passing a hardcoded value of `true` to super

Reply:
tbh not sure why I did that